starter_code/index.html

```diff
@@ -5,22 +5,98 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Hello</title>
 
+		<link rel="stylesheet" 
+		href="https://www.google.com/fonts#UsePlace:use/Collection:Open+Sans">
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
-		<link rel="stylesheet" href="css/style.css">
-
+		<link rel="stylesheet" 
+		href="css/reset.css">
+		<link rel="stylesheet" 
+		href="css/style.css">
 	</head>
+
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
-		</div><!-- #container -->
-	</body>
-</html>
\ No newline at end of file
+			<div class="relaxr-wrapper">
+				
+				<div class="relaxr-nav">
+					<div id="relaxr">Relaxr</div>
+					<div>About</div>
+			 		<div>FAQ</div>
+					<div>Team</div>
+					<div>FAQ</div>
+					<div>Contact Us</div>
+					<div>Blog</div>
+				</div>
+			</div>
+	
+				
+			<div class="div-container">
+
+				<div class="main format">
```

Comment:
@sandyvasquez make sure that when you are naming classes that you are using a - to connect them or it'll read it as two different classes.